pkg/errcode/errcode.go

```diff
@@ -0,0 +1,31 @@
+package errcode
```

Comment:
can you add some short documentation comments above the package so a consumer of this package will know what it is and how to use it ?

example : https://github.com/golang/go/blob/master/src/context/context.go